File: externe/src/EvenementBundle/Entity/Participer.php
<?php

namespace EvenementBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Participer
 *
 * @ORM\Table(name="participer")
 * @ORM\Entity(repositoryClass="EvenementBundle\Repository\ParticiperRepository")
 */
class Participer
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=255)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="idu", type="integer")
     */
    private $idu;

    /**
     * @var int
     *
     * @ORM\Column(name="participate", type="integer")
     */
    private $participate=0;

    /**
     * @var int
     *
     * @ORM\Column(name="ide", type="integer")
     */
    private $ide;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datedeb", type="date")
     */
    private $datedeb;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datefin", type="date")
     */
    private $datefin;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="heuredeb", type="time")
     */
    private $heuredeb;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="heurefin", type="time")
     */
    private $heurefin;

    /**
     * @var string
     *
     * @ORM\Column(name="nomorganisateur", type="string", length=255)
     */
    private $nomorganisateur;

    /**
     * @var string
     *
     * @ORM\Column(name="lieu", type="string", length=255)
     */
    private $lieu;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255)
     */
    private $image;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Participer
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Participer
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set idu
     *
     * @param integer $idu
     *
     * @return Participer
     */
    public function setIdu($idu)
    {
        $this->idu = $idu;

        return $this;
    }

    /**
     * Get idu
     *
     * @return int
     */
    public function getIdu()
    {
        return $this->idu;
    }

    /**
     * Set participate
     *
     * @param integer $participate
     *
     * @return Participer
     */
    public function setParticipate($participate)
    {
        $this->participate = $participate;

        return $this;
    }

    /**
     * Get participate
     *
     * @return int
     */
    public function getParticipate()
    {
        return $this->participate;
    }

    /**
     * Set ide
     *
     * @param integer $ide
     *
     * @return Participer
     */
    public function setIde($ide)
    {
        $this->ide = $ide;

        return $this;
    }

    /**
     * Get ide
     *
     * @return int
     */
    public function getIde()
    {
        return $this->ide;
    }

    /**
     * Set datedeb
     *
     * @param \DateTime $datedeb
     *
     * @return Participer
     */
    public function setDatedeb($datedeb)
    {
        $this->datedeb = $datedeb;

        return $this;
    }

    /**
     * Get datedeb
     *
     * @return \DateTime
     */
    public function getDatedeb()
    {
        return $this->datedeb;
    }

    /**
     * Set datefin
     *
     * @param \DateTime $datefin
     *
     * @return Participer
     */
    public function setDatefin($datefin)
    {
        $this->datefin = $datefin;

        return $this;
    }

    /**
     * Get datefin
     *
     * @return \DateTime
     */
    public function getDatefin()
    {
        return $this->datefin;
    }

    /**
     * Set heuredeb
     *
     * @param \DateTime $heuredeb
     *
     * @return Participer
     */
    public function setHeuredeb($heuredeb)
    {
        $this->heuredeb = $heuredeb;

        return $this;
    }

    /**
     * Get heuredeb
     *
     * @return \DateTime
     */
    public function getHeuredeb()
    {
        return $this->heuredeb;
    }

    /**
     * Set heurefin
     *
     * @param \DateTime $heurefin
     *
     * @return Participer
     */
    public function setHeurefin($heurefin)
    {
        $this->heurefin = $heurefin;

        return $this;
    }

    /**
     * Get heurefin
     *
     * @return \DateTime
     */
    public function getHeurefin()
    {
        return $this->heurefin;
    }

    /**
     * Set nomorganisateur
     *
     * @param string $nomorganisateur
     *
     * @return Participer
     */
    public function setNomorganisateur($nomorganisateur)
    {
        $this->nomorganisateur = $nomorganisateur;

        return $this;
    }

    /**
     * Get nomorganisateur
     *
     * @return string
     */
    public function getNomorganisateur()
    {
        return $this->nomorganisateur;
    }

    /**
     * Set lieu
     *
     * @param string $lieu
     *
     * @return Participer
     */
    public function setLieu($lieu)
    {
        $this->lieu = $lieu;

        return $this;
    }

    /**
     * Get lieu
     *
     * @return string
     */
    public function getLieu()
    {
        return $this->lieu;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return Participer
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }
}

File: tesst/src/entities/participer.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.util.Date;

/**
 *
 * @author AHMED
 */
public class participer {
    
    
    
      private int id ; 
    private String nom;
    private String description;

//    private java.util.Date datedeb = new java.util.Date();
//    private java.util.Date datefin = new java.util.Date();
//    
    
    public java.util.Date datedeb;
    private java.util.Date date_fin;
    private String lieu ;
    private String lien ;
    private String image ;
    private String nom_organisateur;
    private int ide;
    private int idu; 

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
  public String getLien() {
        return lien;
    }

    public void setLien(String lien) {
        this.lien = lien;
    }

    public Date getDatedeb() {
        return datedeb;
    }

    public void setDatedeb(Date datedeb) {
        this.datedeb = datedeb;
    }

    public Date getDate_fin() {
        return date_fin;
    }

    public void setDate_fin(Date date_fin) {
        this.date_fin = date_fin;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getNom_organisateur() {
        return nom_organisateur;
    }

    public void setNom_organisateur(String nom_organisateur) {
        this.nom_organisateur = nom_organisateur;
    }

    public int getIde() {
        return ide;
    }

    public void setIde(int ide) {
        this.ide = ide;
    }

    public int getIdu() {
        return idu;
    }

    public void setIdu(int idu) {
        this.idu = idu;
    }

    
    
    
     
    
}
